devtools/check_contracts_fast.sh

```diff
@@ -6,6 +6,8 @@ for contract_dir in contracts/*/; do
   (
     cd "$contract_dir"
     cargo fmt
+    mkdir -p target/wasm32-unknown-unknown/release/
+    touch target/wasm32-unknown-unknown/release/"$(basename "$contract_dir" | tr - _)".wasm
```

Comment:
Nice. I was thinking about this too but then did not have a great idea how to get the contract name